core/lspserver.py

```diff
@@ -500,6 +511,7 @@ def handle_workspace_configuration_request(self, name, request_id, params):
             sessionSettings = settings.get(section, {})
 
             if self.server_info["name"] == "vscode-eslint-language-server":
+                sessionSettings = copy.copy(settings)
```

Comment:
why need copy

Reply:
Not sure if somewhere else will modify workspace config, resulting in changing the server info secretly. It is just defensive — we can remove it if not needed.